bedrock/privacy/templates/privacy/index.html

```diff
@@ -12,7 +12,7 @@
 {% block body_class %}mzp-t-mozilla format-none{% endblock%}
 {% block body_id %}privacy-landing{% endblock %}
 
-{% block article_header_logo %}{{ static('img/trademarks/logo-mozm.png') }}{% endblock %}
+{% block article_header_logo %}{{ static('img/trademarks/symbol.svg') }}{% endblock %}
```

Comment:
It was suggested to use the green flag with black background here, but I've gone with the asset that exists in the `trademarks` folder for simplicity (https://www.mozilla.org/en-US/foundation/trademarks/list/)

Reply:
I'm not sure who the authority is on this - but I like what you've done.